### PR TITLE
Adds bazel project opencensus-cpp

### PR DESCRIPTION
**Pull request by catenacyber**
@g-easy would you be interested in continuous fuzzing for opencensus-cpp ?
This PR enables it with oss-fuzz with your fuzz targets.
It changes a bit the Basel build system to use https://github.com/bazelbuild/rules_fuzzing and https://google.github.io/oss-fuzz/getting-started/new-project-guide/bazel/ cf 

cc @stefanbucur could `cc_fuzz_test` support corpus like it is used here https://github.com/census-instrumentation/opencensus-cpp/blob/ba631066779a534267fdb1321b19850eb2b0c000/opencensus/tags/BUILD#L194

### CODE REVIEW

**Comment by stefanbucur** [comment]
> cc @stefanbucur could `cc_fuzz_test` support corpus like it is used here https://github.com/census-instrumentation/opencensus-cpp/blob/ba631066779a534267fdb1321b19850eb2b0c000/opencensus/tags/BUILD#L194

Yes, `cc_fuzz_test` already supports the `corpus` attribute in the same way: https://github.com/bazelbuild/rules_fuzzing/blob/master/docs/guide.md#specifying-seed-corpora

**Comment by g-easy** [comment]
Nice! Thanks for taking care of this.

**Comment by g-easy** [comment]
@jsuereth - what would you suggest re: email address?

**Comment by jsuereth** [comment]
sorry for delay, was out for a bit.   opentelemetry-dev+opencensus-fuzz@google.com would be ideal for us.

File: projects/opencensus-cpp/project.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+homepage: "https://opencensus.io/"
+language: c++
+primary_contact: "opentelemetry-dev+opencensus-fuzz@google.com"
```
Comment (oliverchang) [inline projects/opencensus-cpp/project.yaml:3]: Unfortunately the "+" breaks the auth for ClusterFuzz here, even if this was an email associated with an individual google account. In this case though this looks like a mailing list so it's unlikely to work anyway.

@jsuereth could you please suggest other individual emails to add here? Filtering these emails should be easy.

**Review by oliverchang — APPROVED** [review]
LGTM with a comment about the email

**Comment by asraa** [comment]
friendly ping @jsuereth

File: projects/opencensus-cpp/project.yaml (hunk 1 above)
Comment (jsuereth) [reply projects/opencensus-cpp/project.yaml:3]: You can use joshuasuereth@google.com for now then.

**Pull request merged**